examples/api/observablehq.config.js

```diff
@@ -1,8 +1,11 @@
 export default {
   root: "src",
-  title: "Analyzing web logs",
+
+  // Shared Observable example configuration; feel free to remove this.
+  title: "Observable Framework",
   pager: false,
   toc: false,
+  sidebar: false,
```

Comment:
yes the wrapper would solve a lot of the issues mentioned already: indexing, sharing deploy ids, centralizing the custom css, analytics, and other config options that are only needed for our deploys and otherwise add cognitive load for users who want to fork/copy the code.

Reply:
Yes, good future work.